docs/en/integrations/data-ingestion/data-formats/json.md

```diff
@@ -0,0 +1,576 @@
+---
+sidebar_label: JSON
+sidebar_position: 2
+slug: /en/integrations/data-formats/json
+---
+
+# Importing and exporting JSON data in ClickHouse
+
+JSON is a popular format for exchanging data between different layers of modern applications. ClickHouse provides many tuning options to support almost any form of JSON data.
+
+
+## Importing JSON data
+
+To import JSON data, we first have to define which JSON type to use. This will depend on how the input data is structured.
+
+### Importing from an array of JSON objects
+
+One of the most popular forms of JSON data is having a list of JSON objects in a JSON array, like in [this example](assets/list.json):
+
+
+```bash
+> cat list.json
+[
+  {
+    "path": "Akiba_Hebrew_Academy",
+    "month": "2017-08-01",
+    "hits": 241
+  },
+  {
+    "path": "Aegithina_tiphia",
+    "month": "2018-02-01",
+    "hits": 34
+  },
+  ...
+]
+```
+
+Let’s create a table for this kind of data:
+
+```sql
+CREATE TABLE sometable
+(
+    `path` String,
+    `month` Date,
+    `hits` UInt32
+)
+ENGINE = MergeTree
+ORDER BY tuple(month, path)
+```
+
+To import a list of JSON objects, we can use a [JSONEachRow](/docs/en/interfaces/formats.md/#jsoneachrow) format:
+
+```sql
+INSERT INTO sometable
+FROM INFILE 'list.json'
```

Comment:
this is available as an asset but It needs the user to find it... @DanRoscigno do we link to this or should we provide download links for the files?

Reply:
added a link in a sentence right before the code block